wcfsetup/install/files/lib/system/database/table/index/DatabaseTableForeignKey.class.php

```diff
@@ -14,45 +14,41 @@
  * @package WoltLabSuite\Core\System\Database\Table\Index
  * @since   5.2
  */
-class DatabaseTableForeignKey
```

Comment:
Please drop the commit with the changes to this file. These are larger than I'm comfortable with and I want to have a look whether the types actually even make sense in the first place.